Client/app/src/main/java/at/aau/anti_mon/client/Lobby.java

```diff
@@ -40,4 +39,9 @@ public void onCancelLobby(View view) {
         Intent intent = new Intent(Lobby.this, Start_new_Game.class);
         startActivity(intent);
     }
+
+    public void onStartGame(View view) {
```

Comment:
Die Klasse musste dringend angepasst werden ( Issue#14) und heißt nun LobbyActivity.  
Die Methode kann jedoch einfach in die neue Klasse hinzugefügt werden.